frontend/packages/process-editor/src/components/Canvas/Canvas.module.css

```diff
@@ -15,7 +19,35 @@
   font-weight: 500;
 }
 
-.alertContainer  {
+.alertContainer {
   composes: container;
   max-width: 780px;
-}
\ No newline at end of file
+}
+
+div[class*='bpmn-icon-task-generic'] {
+  background-color: black;
+}
+
+div[class*='bpmn-icon-task-generic']:hover {
+  background-color: #0086e6;
```

Comment:
Should we use the colours from the Altinn brand with design tokens? 😄 https://www.designsystemet.no/grunnleggende/designelementer/farger

Reply:
This color comes from the BMPN editor - I think it's better to be consistent with the colors in the palette, instead of trying to hack an override of those colors in our css 🤔 Although it would probably be a good idea to actually use the color variables that the bmpnjs package uses then 😄

Reply:
I agree! 👏